charter/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Font and Text Community Group Charter
+---
+This charter:  {{ page.url | remove: "index.html" | prepend: site.url }}
+
+Feedback about this document:
+    https://github.com/font-text/font-text.github.io/issues
+
+## Scope
+
+The Font and Text community group gathers individuals and organisations
+interested in discussing and developing specifications and implementations
+for technologies such as shaping and layout
+which operate on and at the interface between
+text encoding and font formats.
+
+Under the present charter, the Community Group will
+determine a structure and process
+under which such specifications may be created
+by which the greatest number of interested parties may be able to participate.
+Until such a structure and process is determined,
+no technical development of specifications or implementations
+will be undertaken by this Community Group
+and no technical contributions will be accepted.
+
+## Deliverables
+
+* This Community Group aims to produce a proposed Charter under which
+this community should conduct its technical work,
+either to be used as an updated Charter for this CG,
+or to set up a different group,
+possibly a W3C Working Group,
+to conduct that work.
+
+* If the Community Groups determines that it would be appropriate
+to divide the technical work into several venues,
+it may produce several such proposed charters
+rather than a single one.
+
+* Various supporting documents and articles
+describing the problem space
+may also be produced.
+
+All deliverables of this Community Group will use the
+[W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
+
+This group maintains an [index](https://font-text.github.io/work-items/)
+of the documents it is working on.
+
+## Communication
+
+This group operates under the
+[W3C Code of Ethics and Professional Conduct ](https://www.w3.org/Consortium/cepc/).
+
+This group will conduct its work publicly,
+primarily using the public-font-text@w3.org mailing list
+([archives](https://lists.w3.org/Archives/Public/public-font-text/)
+for minutes, announcements, and high level discussions,
+and [GitHub](https://github.com/font-text)
+for detailed discussions
+(See [Contribution Mechanics](#contribution-mechanics).
+
+Teleconferences and face-to-face meetings
+may be held from time to time,
+provided sufficient advance notice is given.
+Minutes of any such meeting will be
+posted to the group’s public mailing list
+in a timely manner,
+and any resolution will be called out explicitly
+(See [Decision Policy](#decision-policy)).
+
+Deliverables and supporting documents
+produced by this Community Group are
+hosted on the group’s [Github](https://github.com/font-text).
```

Comment:
`Github` should be `GitHub` throughout this document.

Reply:
```suggestion
hosted on the group’s [GitHub](https://github.com/font-text).
```